### DISCUSSION TITLE
OTLP Ingestion - Is it OTLP http that you accept or OTLP grpc? Or both?

### DISCUSSION THREAD

**Discussion by arukiidou** [prometheus/prometheus#12617] [Q&A] [answered]
- https://github.com/prometheus/prometheus/pull/12571

@gouthamve

- Is it OTLP accepts?
  - [otlpmetrichttp](https://github.com/open-telemetry/opentelemetry-go/tree/main/exporters/otlp/otlpmetric/otlpmetrichttp)
  - [otlpmetricgrpc](https://github.com/open-telemetry/opentelemetry-go/tree/main/exporters/otlp/otlpmetric/otlpmetricgrpc)

**Comment by gouthamve** [comment, marked as answer]
We do only OTLP HTTP for now.

We are open to adding grpc ingestion in the future.
